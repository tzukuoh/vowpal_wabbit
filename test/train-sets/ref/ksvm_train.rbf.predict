0
0.149683
0.007743
-0.139958
-0.239910
-0.294489
-0.221856
-0.276376
-0.350765
-0.402906
-0.318881
-0.380436
-0.444594
-0.479524
-0.460967
-0.392380
-0.385918
-0.339858
-0.425975
-0.429099
-0.486633
-0.418433
-0.369563
-0.351008
-0.290364
-0.404112
-0.438487
-0.526117
-0.465054
-0.397440
-0.475374
-0.365495
-0.436116
-0.488418
-0.474721
-0.434405
-0.448698
-0.429734
-0.430610
-0.280532
-0.318512
-0.377771
-0.317385
-0.346526
-0.464305
-0.513655
-0.435050
-0.443046
-0.522330
-0.434429
-0.478220
-0.398383
-0.383485
-0.451203
-0.436492
-0.284591
-0.434623
-0.488784
-0.496739
-0.433066
-0.403163
-0.356238
-0.241399
-0.582897
-0.415821
-0.424499
-0.288895
-0.413035
-0.564809
-0.493150
-0.355952
-0.259809
-0.383713
-0.397731
-0.244233
-0.317643
-0.323920
-0.465489
-0.435307
-0.473179
-0.453149
-0.448030
-0.315877
-0.413427
-0.522044
-0.529590
-0.500524
-0.381069
-0.340907
-0.590603
-0.413271
-0.445645
-0.148096
-0.322534
-0.257272
-0.467063
-0.391364
-0.266517
-0.445710
-0.379755
-0.301372
-0.331075
-0.310802
-0.353114
-0.300292
-0.315821
-0.348095
-0.468289
-0.421367
-0.340647
-0.257080
-0.249754
-0.339345
-0.422768
-0.184818
-0.422488
-0.404505
-0.340865
-0.364052
-0.408904
-0.363454
-0.326794
-0.275695
-0.424731
-0.229625
-0.116087
-0.272610
-0.226703
-0.469335
-0.304120
-0.274133
-0.313477
-0.258307
-0.326997
-0.326560
-0.442960
-0.295745
-0.231223
0.110537
-0.297293
-0.415252
-0.525797
-0.005597
0.275089
-0.224615
-0.038253
-0.220323
0.141756
-0.438874
-0.308143
-0.045118
-0.152948
0.002146
-0.235369
-0.410130
-0.128069
0.254856
-0.098371
0.231093
-0.161466
0.474805
-0.367932
-0.395389
-0.070414
-0.204446
-0.199197
0.221601
-0.330751
-0.325514
0.213292
0.173295
-0.300682
-0.290757
-0.016528
-0.278284
-0.181906
-0.583235
0.165977
-0.055550
0.213615
-0.054914
0.025768
-0.032288
-0.326226
-0.275042
-0.227303
-0.127785
-0.364131
-0.320264
-0.016729
0.259401
-0.083472
-0.223162
-0.255252
-0.254169
-0.229441
-0.091872
-0.308170
-0.177985
-0.189094
-0.226706
-0.038350
0.000802
-0.257037
-0.316039
-0.376949
0.024301
-0.128445
-0.232979
-0.153235
-0.195607
-0.435049
-0.403177
-0.177024
0.833131
-0.433451
-0.210849
-0.337157
-0.282193
-0.328782
-0.209215
-0.496356
-0.347661
-0.342635
-0.132809
-0.264901
0.882256
0.173201
-0.254508
-0.410941
-0.508959
-0.547308
-0.515769
-0.258538
-0.077837
-0.423000
-0.430809
0.012972
0.536660
-0.091294
-0.013880
-0.235773
-0.504134
0.245609
-0.191421
-0.044976
0.045034
-0.272412
-0.516458
0.129464
