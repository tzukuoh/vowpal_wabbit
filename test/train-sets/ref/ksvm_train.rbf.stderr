using l2 regularization = 1
predictions = ksvm_train.rbf.predict
Lambda = 1
Kernel = rbf
bandwidth = 1
Num weight bits = 18
learning rate = 0.5
initial_t = 0
power_t = 0.5
using no cache
Reading datafile = train-sets/rcv1_smaller.dat
num sources = 1
average  since         example        example  current  current  current
loss     last          counter         weight    label  predict features
1.000000 1.000000            1            1.0   1.0000   0.0000       50
1.074842 1.149683            2            2.0  -1.0000   0.1497      103
1.004367 0.933892            4            4.0  -1.0000  -0.1400      134
0.946727 0.889087            8            8.0  -1.0000  -0.2764      145
0.928492 0.910257           16           16.0   1.0000  -0.3924       23
0.920368 0.912244           32           32.0  -1.0000  -0.3655       31
0.905693 0.891018           64           64.0  -1.0000  -0.5829       60
0.928926 0.952158          128          128.0   1.0000  -0.2267      105

finished run
number of examples = 250
weighted example sum = 250.000000
weighted label sum = -22.000000
average loss = 0.893457
best constant = -0.088000
best constant's loss = 0.992256
total feature number = 19870
Num support = 250
Number of kernel evaluations = 186885 Number of cache queries = 95668
Total loss = 223.364304
Done freeing model
Done freeing kernel params
Done with finish 
